src/pages/workspace/WorkspaceMoreFeaturesPage.tsx

```diff
@@ -111,6 +113,34 @@ function WorkspaceMoreFeaturesPage({policy, route}: WorkspaceMoreFeaturesPagePro
                 setIsDisableExpensifyCardWarningModalOpen(true);
             },
         });
+        spendItems.splice(2, 0, {
+            icon: Illustrations.CompanyCard,
+            titleTranslationKey: 'workspace.moreFeatures.companyCards.title',
+            subtitleTranslationKey: 'workspace.moreFeatures.companyCards.subtitle',
+            isActive: policy?.areCompanyCardsEnabled ?? false,
+            pendingAction: policy?.pendingFields?.areCompanyCardsEnabled,
+            disabled: !isEmptyObject(companyCardsList),
+            action: (isEnabled: boolean) => {
+                if (!policyID) {
```

Comment:
How could we be here and not have policyID?

Reply:
It's probably question for how is withPolicyAndFullscreenLoading HOK done. If we remove optional chaining - we will get TS error
<img width="647" alt="Screenshot 2024-08-23 at 16 28 32" src="https://github.com/user-attachments/assets/f00015ba-ef6d-43e7-a339-2e70ca3c6846">

Reply:
Why would that component define policy as optional?

Reply:
Not sure what to answer here - optional policy is not defined in this code - and was historically here

Reply:
@iwiznia i think you can check this discussion https://github.com/Expensify/App/pull/44549#discussion_r1693555324

Reply:
hmmm ok, seems odd to me but out of scope